README.md

```diff
@@ -1295,7 +1295,7 @@ Com o objetivo de alcançar uma abrangência maior e encorajar mais pessoas a co
                 </a>
             </td>
             <td> <!-- Java -->
-                <a href="./src/java/TorreDeHanoi.java">
+                <a href="./src/java/TowersOfHanoi.java">
```

Comment:
wdyt about using `TowerOfHanoi`? (singular instead of plural)

Reply:
This seems to be the correct title of the algorithm, I'll update this as well as fixing that last linker error tonight. Thanks for the review!!!